src/js/components/container/Timer.jsx

```diff
@@ -44,10 +44,23 @@ class Timer extends Component {
     })
   }
 
-  render() {
-  	return <Modal isModal size="tiny"
-    open={this.state.running} closeStyle={{'display': 'none'}}>
-      <div className="text-center h1">{this.state.time}</div>
+  render () {
+    const overlayStyle = {
+      'backgroundColor': 'rgba(0,0,0,0)'

```

Comment:
Why is this still here and not in the matching CSS file?

Reply:
I could nit add class/id to the modal. The only way I found is to pass Jsons to the modal it self.
https://www.npmjs.com/package/react-foundation-modal#props

Reply:
Fine. It'll be fine when we user semantic